### PR TITLE
#93 - add Wind provider

### LINKED ISSUE #93: Add Wind provider
Site: https://wind.yango.com/official-website/#/rider?lang=global

### PR DESCRIPTION
**Pull request by BarTracz**
Made some adjustments to providers related to Türkiye and added Wind. This should close #93

### CODE REVIEW

File: app/Importers/WindDataImporter.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Importers;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Symfony\Component\DomCrawler\Crawler;
+
+class WindDataImporter extends DataImporter
```
Comment (mtracz) [inline app/Importers/WindDataImporter.php:10]: It would be good to write tests for this importer.

Comment (BarTracz) [reply app/Importers/WindDataImporter.php:10]: How this test should look like, and what it should test?

File: app/Importers/WindDataImporter.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            foreach ($section->childNodes as $node) {
+                if ($node->nodeName === "script" && $node->hasAttribute("src")) {
+                    $scriptSrc = $node->getAttribute("src");
+                }
+            }
+        }
+        $jsSrc = "https://wind.yango.com/official-website/" . $scriptSrc;
+        $jsContent = file_get_contents($jsSrc);
+
+        $subtring_start = strpos($jsContent, "YOUR SECRET SUPERPOWER?");
```
Comment (AleksandraKozubal) [inline app/Importers/WindDataImporter.php:50]: Did you mean `$substring_start` ?

File: app/Importers/WindDataImporter.php (hunk 1 above)
Comment (AleksandraKozubal) [reply app/Importers/WindDataImporter.php:10]: My suggestion is to leave this test for later - after yesterday's conversation, I agreed that I would test each provider

Comment (mtracz) [reply app/Importers/WindDataImporter.php:10]: As you wish, but create draft PR with at least one importer tested, and ping me or someone else for a review.
Suggested way of creating new features is to write tests for them in the same PR.

**Review by mtracz — APPROVED** [review]

File: app/Importers/WindDataImporter.php (hunk 2 above)
Comment (BarTracz) [reply app/Importers/WindDataImporter.php]: yes, fixed now

**Review by AleksandraKozubal — APPROVED** [review]

**Pull request merged**